dev/fattest.simplicity/src/componenttest/topology/utils/tck/TCKResultsWriter.java

```diff
@@ -77,43 +78,42 @@ public static void preparePublicationFile(TCKResultsInfo resultInfo, String[] sp
                                           openLibertyVersion,
                                           "]\n\n",
-                                          "* Specification Name, Version and download URL:\n+\n",
+                                          "* Specification Name, Version and download URL:\n\n",
                                           "link:https://jakarta.ee/specifications/", specName, "/", specVersion, "[Jakarta EE ", specName, " ", specVersion, "]\n\n",
-                                          "* TCK Version, digital SHA-256 fingerprint and download URL:\n+\n",
+                                          "* TCK Version, digital SHA-256 fingerprint and download URL:\n\n",
                                           "https://download.eclipse.org/jakartaee/", ".zip[Jakarta EE]\n",
                                           "SHA-256: ``\n\n",
-                                          "* Public URL of TCK Results Summary:\n+\n",
-                                          "link:", openLibertyVersion, "-Java", javaMajorVersion, "-TCKResults.html[TCK results summary]\n\n",
+                                          "* Public URL of TCK Results Summary:\n\n",
+                                          "link:", filename, "[TCK results summary]\n\n",
                                           "* Any Additional Specification Certification Requirements:\n\n",
-                                          "+\nJakarta Contexts and Dependency Injection\n",
+                                          "\nJakarta Contexts and Dependency Injection\n",
                                           "link:https://download.eclipse.org/jakartaee/cdi/", "SHA-256:\n``\n",
-                                          "+\nJakarta Bean Validation\n",
+                                          "\nJakarta Bean Validation\n",
                                           "link:https://download.eclipse.org/jakartaee/bean-validation/", "SHA-256:\n``\n",
-                                          "+\nDependency Injection\n",
+                                          "\nDependency Injection\n",
                                           "link:https://download.eclipse.org/jakartaee/dependency-injection/", "SHA-256:\n``\n",
-                                          "+\nDebugging\n",
+                                          "\nDebugging\n",
                                           "link:https://download.eclipse.org/jakartaee/debugging/", "SHA-256:\n``\n",
-                                          "* Java runtime used to run the implementation:\n+\n", javaVersion,
-                                          "\n\n* Summary of the information for the certification environment, operating system, cloud, ...:\n+\n",
+                                          "* Java runtime used to run the implementation:\n\n", javaVersion,
+                                          "\n\n* Summary of the information for the certification environment, operating system, cloud, ...:\n\n",
```

Comment:
You've removed a bunch of `+` characters in this section.

I think those were necessary to keep the text from multiple lines attached to the same bullet point. See [list continuations](https://docs.asciidoctor.org/asciidoc/latest/lists/continuation/#list-continuation)

As an aside, this whole section with an array of document sections is nasty. It would probably be more readable if it was multiple calls to a `StringBuilder`.

Reply:
When I ran this locally and looked at a preview view for these files, those `+` chars were showing up all over the place. I was intending to get rid of these String[]s in a follow on PR anyway.

Reply:
Odd. They are in [existing published files](https://raw.githubusercontent.com/OpenLiberty/certifications/prod/microprofile/metrics/3.0/21.0.0.1-cl210120201202-1100-TCKResults.adoc) and do [appear to render correctly](https://openliberty.io/certifications/microprofile/metrics/3.0/21.0.0.1-cl210120201202-1100-TCKResults.html).